viewer/src/setupProxy.js

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+const http = require("http");
+const https = require("https");
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+// Using keepAlive drastically improves the response times for this proxy,
+// especially when being hammered with many requests at once such as the initial
+// load of the viewer.
+// https://github.com/nodejitsu/node-http-proxy/issues/1058
+const httpAgent = new http.Agent({ keepAlive: true });
+const httpsAgent = new https.Agent({ keepAlive: true });
+
+const gwvUrl = process.env.REACT_APP_GWV_URL;
```

Comment:
Minor gripe but we shouldn't be using the acronym "GWV" at all anymore.

Reply:
No we really shouldn't :stuck_out_tongue:. Was a copy over from the Designer.

Reply:
Changed to `viewerUrl` and `REACT_APP_VIEWER_URL`